### PR TITLE
Add D7SMS Connector

### PR DESCRIPTION
**Pull request by d7networks**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by d7networks** [comment]
Added Readme file

File: connectors/D7SMS/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "capabilities": [],
```
Comment (ShyamSunder90) [inline connectors/D7SMS/apiProperties.json]: Can you please update this to : "capabilities": [ "actions" ],

File: connectors/D7SMS/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                "content": {
+                  "description": "content",
+                  "type": "string"
+                },
+                "from": {
+                  "description": "from",
+                  "type": "string"
+                },
+                "to": {
+                  "default": 971562316353,
```
Comment (ShyamSunder90) [inline connectors/D7SMS/apiDefinition.swagger.json]: Can you please remove this default value?

**Comment by ShyamSunder90** [comment]
> Added Readme file

Thank you!

File: connectors/D7SMS/apiProperties.json (hunk 1 above)
Comment (d7networks) [reply connectors/D7SMS/apiProperties.json]: Done

File: connectors/D7SMS/apiDefinition.swagger.json (hunk 2 above)
Comment (d7networks) [reply connectors/D7SMS/apiDefinition.swagger.json]: Done

**Pull request merged**